naming/etcd/etcd.go

```diff
@@ -0,0 +1,145 @@
+package etcd 
+
+import (
+	"sync"
+	"log"
+
+	etcdcl "github.com/coreos/etcd/client"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc/naming"
+)
+
+type kv struct {
+	key, value string
+}
+
+// recvBuffer is an unbounded channel of *kv.
```

Comment:
// recvBuffer is an unbounded channel of *kv to record all the pending changes from etcd server.